app/app.py

```diff
@@ -149,7 +149,7 @@ def get_trip_features(intersect_ids, grid, flow, mode):
     CORS(app)
 
 
-@app.get("/v1/trips")
+@app.get("/trips", version=1)
```

Comment:
Does this change the endpoints when need to use in the client-side app?

Docs suggest no: https://sanic.readthedocs.io/en/latest/sanic/versioning.html#versioning

Reply:
It does not.